UI/Selenium/Pages/BookingListPage.cs

```diff
@@ -11,8 +11,10 @@ public class BookingListPage
         public static By VideoHearingsTable => By.Id("vh-table");
         public static By HearingDateTitle => By.XPath($"//div[text()[contains(.,'{DateTime.Today.ToString("dd MMMM yyyy")}')]]");
         public static By HearingDetailsRow => By.XPath("//div[@class='vh-row-created']//div[@class='govuk-grid-row vh-row vh-a']");
-        public static By HearingDetailsRowSpecific(string text) => By.XPath($"//div[text()[contains(.,'{text}')]]");
+        public static By HearingDetailsRowSpecific(string caseNumber) => By.XPath($"//div[text()[contains(.,'{caseNumber}')]]");
+        public static By HearingSelectionSpecificRow(string caseNumber) => By.XPath($"//div[@class='govuk-grid-row vh-row vh-a' and contains(.,'{caseNumber}')]//div[@class='vh-created-booking']");
```

Comment:
Same issues here with the xpath where I would prefer an id or something very clear.

Reply:
BookingListPage  lists a number of cases. So we need to check for a row that contains a specific Case number.